### ISSUE TITLE
Request to add your own bitcoin amount. 

### ISSUE THREAD

**Issue by Dan1jel** [Brutuski/tmux-crypto#2]
Would be nice to add your own bitcoin amount to you can see what your amount is worth atm. 

Ex. `#{btc_kr=0.028429}` is `BTC: 950kr`.

**Comment by Brutuski** [comment]
Yup, will be working on this and will try to get this feature out next.
